satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -0,0 +1,129 @@
+reader:
+  name: seviri_l2_grib
+  short_name: SEVIRI L2 GRIB
+  long_name: MSG SEVIRI L2 (GRIB)
+  description: Reader for EUMETSAT MSG SEVIRI L2 files in GRIB format.
+  sensors: [seviri]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
+file_types:
+  # EUMETSAT MSG SEVIRI L2 Cloud Mask files in GRIB format
+  grib_seviri_clm:
+    file_reader: !!python/name:satpy.readers.seviri_l2_grib.SeviriL2GribFileHandler
+    file_patterns:
+        - 'CLMEncProd_{start_time:%Y%m%d%H%M%S}Z_00_{server:8s}_{spacecraft:5s}_{scan_mode:3s}_{sub_sat:5s}'
+        - '{spacecraft:4s}-SEVI-MSGCLMK-0100-0100-{start_time:%Y%m%d%H%M%S}.000000000Z-{product_creation_time:%Y%m%d%H%M%S}-{ord_num:7s}'
+        - '{spacecraft:4s}-SEVI-MSGCLMK-0100-0100-{start_time:%Y%m%d%H%M%S}.000000000Z-{product_creation_time:%Y%m%d%H%M%S}-{ord_num:7s}.grb'
+
+  # EUMETSAT MSG SEVIRI L2 Optimal Cloud Analysis files in GRIB format
+  grib_seviri_oca:
+    file_reader: !!python/name:satpy.readers.seviri_l2_grib.SeviriL2GribFileHandler
+    file_patterns:
+        - 'OCAEncProd_{start_time:%Y%m%d%H%M%S}Z_00_{server:8s}_{spacecraft:5s}_{scan_mode:3s}_{sub_sat:5s}'
+        - '{spacecraft:4s}-SEVI-MSGOCAE-0100-0100-{start_time:%Y%m%d%H%M%S}.000000000Z-{product_creation_time:%Y%m%d%H%M%S}-{ord_num:7s}'
+        - '{spacecraft:4s}-SEVI-MSGOCAE-0100-0100-{start_time:%Y%m%d%H%M%S}.000000000Z-{product_creation_time:%Y%m%d%H%M%S}-{ord_num:7s}.grb'
+datasets:
+
+  cloud_mask:
+    name: cloud_mask
+    resolution: 3000
+    file_type: grib_seviri_clm
+    parameter_number: 7
+    units: ""
+    standard_name: cloud_mask_classification
```

Comment:
I couldn't find that in the CF standard name table (same for `scene_classification` and `cost_function`). Maybe use `long_name` instead?

Reply:
names not found in CF table changed to `long_name`.